.changeset/popular-carrots-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Fixed SubmenuSubtitle is always white including on light background
```

Comment:
```suggestion
Fixed an issue causing `SidebarSubmenu` text to not follow the theme color
```

Reply:
Thank you for the suggestion! I updated the changeset according to your suggestion.